app/code/ZP/LoyaltyProgram/Setup/Patch/Data/AddBasicPrograms.php

```diff
@@ -37,6 +37,64 @@ public function apply(): DataPatchInterface
             ]
         );
 
+        $connection->insert(
+            LoyaltyProgramInterface::MAIN_TABLE,
+            [
+                LoyaltyProgramInterface::PROGRAM_ID => 3,
+                LoyaltyProgramInterface::PROGRAM_NAME => 'Bronze',
```

Comment:
You cannot just make changes in the data patch after this patch has been pushed with a specific version (git tag). This case it means this patch was applied and already exists in the database table 'patch_list' . Please use separata data patch and specify previous patch in the dependencies in order to be sure your patch gets executed after the first patch.